dev/kyuubi-extension-spark-3-1/src/test/resources/sql-classification-default.json

```diff
@@ -1,34 +1,60 @@
 {
   "AlterDatabasePropertiesCommand": "ddl",
-  "AlterTableRenameCommand": "ddl",
-  "AlterTableRenamePartitionCommand": "ddl",
   "AlterTableAddColumnsCommand": "ddl",
-  "AlterTableChangeColumnCommand": "ddl",
   "AlterTableAddPartitionCommand": "ddl",
+  "AlterTableChangeColumnCommand": "ddl",
   "AlterTableDropPartitionCommand": "ddl",
-  "AlterTableSetPropertiesCommand": "ddl",
-  "AlterTableSetLocationCommand": "ddl",
-  "AlterTableSerDePropertiesCommand": "ddl",
-  "AlterTableUnsetPropertiesCommand": "ddl",
+  "AlterTableRecoverPartitionsCommand": "ddl",
+  "AlterTableRenameCommand": "ddl",
   "AlterTableRenameCommand": "ddl",
+  "AlterTableRenamePartitionCommand": "ddl",
+  "AlterTableSerDePropertiesCommand": "ddl",
+  "AlterTableSetLocationCommand": "ddl",
   "AlterTableSetPropertiesCommand": "ddl",
+  "AlterTableSetPropertiesCommand": "ddl",
+  "AlterTableUnsetPropertiesCommand": "ddl",
   "AlterTableUnsetPropertiesCommand": "ddl",
   "AlterViewAsCommand": "ddl",
+  "CreateDataSourceTableAsSelectCommand": "ddl",
+  "CreateDataSourceTableCommand": "ddl",
   "CreateDatabaseCommand": "ddl",
   "CreateFunctionCommand": "ddl",
-  "CreateDataSourceTableCommand": "ddl",
-  "CreateDataSourceTableAsSelectCommand": "ddl",
   "CreateTableLikeCommand": "ddl",
   "CreateViewCommand": "ddl",
   "DropDatabaseCommand": "ddl",
   "DropFunctionCommand": "ddl",
-  "NoopDropTable": "ddl",
   "DropTableCommand": "ddl",
-  "TruncateTableCommand": "ddl",
-  "AlterTableRecoverPartitionsCommand": "ddl",
+  "NoopDropTable": "ddl",
   "SetCatalogAndNamespace": "ddl",
+  "TruncateTableCommand": "ddl",
+  "InsertIntoDataSourceDirCommand": "dml",
   "InsertIntoHadoopFsRelationCommand": "dml",
+  "InsertIntoHiveDirCommand": "dml",
```

Comment:
insertintodir

Reply:
this grammar is "INSERT OVERWRITE DIRECTORY".
when we got its logicalPlan, the simple name is "InsertIntoDataSourceDirCommand"